src/DotNetLightning.Core/Channel/Channel.fs

```diff
@@ -255,22 +258,23 @@ module Channel =
                 let localParams = state.InitFundee.LocalParams
                 let channelKeys = state.InitFundee.ChannelKeys
                 let localCommitmentSecret = ChannelUtils.buildCommitmentSecret (channelKeys.CommitmentSeed, 0UL)
-                let acceptChannel = { AcceptChannel.TemporaryChannelId = msg.TemporaryChannelId
-                                      DustLimitSatoshis = localParams.DustLimitSatoshis
-                                      MaxHTLCValueInFlightMsat = localParams.MaxHTLCValueInFlightMSat
-                                      ChannelReserveSatoshis = localParams.ChannelReserveSatoshis
-                                      HTLCMinimumMSat = localParams.HTLCMinimumMSat
-                                      MinimumDepth = cs.Config.ChannelHandshakeConfig.MinimumDepth
-                                      ToSelfDelay = localParams.ToSelfDelay
-                                      MaxAcceptedHTLCs = localParams.MaxAcceptedHTLCs
-                                      FundingPubKey = channelKeys.FundingKey.PubKey
-                                      RevocationBasepoint = channelKeys.RevocationBaseKey.PubKey
-                                      PaymentBasepoint = channelKeys.PaymentBaseKey.PubKey
-                                      DelayedPaymentBasepoint = channelKeys.DelayedPaymentBaseKey.PubKey
-                                      HTLCBasepoint = channelKeys.HTLCBaseKey.PubKey
-                                      FirstPerCommitmentPoint = localCommitmentSecret.PubKey
-                                      ShutdownScriptPubKey = cs.Config.ChannelOptions.ShutdownScriptPubKey }
-
+                let acceptChannel: AcceptChannelMsg = {
```

Comment:
I don't think you need to add types, the most important thing is renaming the variable to add the "Msg" suffix, which you forgot to do here

Reply:
To me renaming sounds not necessary, it is not a big deal.

Reply:
I moved the type annotation

```
let acceptChannel = { AcceptChannel.TemporaryChannelId
                      ^^^^^^^^^^^^^
                                   - from here
```

```
let acceptChannel: AcceptChannelMsg {
                   ^^^^^^^^^^^^^^^^
                                   - to here
```

Because another similar line of code in the file required me to (because adding the suffix screwed up the indentation).

Adding the suffix to the variable name is a good idea too though. I'll edit the commit.